### PR TITLE
Fix for DOC-734 changing launchpad links to JIRA, 5.5 version

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: doc/source/upstream-bug-fixes.rst [hunk 1]
```diff
@@ -4,645 +4,651 @@
  List of upstream |MySQL| bugs fixed in |Percona Server| 5.5
 =============================================================
 
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`53588` - Blackhole : Specified key was too long; max key length is 1000 bytes      |
+|:JIRA bug: :psbug:`1126`                                                                                     |
+|:Upstream state: Verified (checked on 2018-01-30)                                                            |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: Why the check date is so old?

File: doc/source/upstream-bug-fixes.rst [hunk 2]
```diff
@@ -4,645 +4,651 @@
...
+|:Fix Released: :rn:`5.5.59-38.11`                                                                            |
+|:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83264` - uint3korr should stop reading four instead of three bytes on x86          |
-|:Launchpad bug: :bug:`1628417`                                                                               |
+|:JIRA bug: :psbug:`3567`                                                                                     |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: Please avoid extra whitespace at the end of the line (it seems that "JIRA bug" in this PR has it)

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/upstream-bug-fixes.rst (hunk 2 above)
Comment (laurynas-biveinis) [reply doc/source/upstream-bug-fixes.rst]: The comment still stands

File: doc/source/upstream-bug-fixes.rst [hunk 3]
```diff
@@ -4,645 +4,651 @@
...
+|:JIRA bug: :psbug:`1076`                                                                                     |
+|:Upstream state: Verified (checked on 2018-05-24)                                                            |
 |:Fix Released: :rn:`5.5.57-38.9`                                                                             |
 |:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83814` - Add support for OpenSSL 1.1                                               |
-|:Launchpad bug: :bug:`1702903`                                                                               |
-|:Upstream state: Verified (checked on 2017-08-11)                                                            |
+|:JIRA bug: :psbug:`1105`                                                                                     |
+|:Upstream state: Closed                                                                                      |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: Please avoid whitespace at the end of the line for "Upstream state" too

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/upstream-bug-fixes.rst (hunk 2 above)
Comment (fiowro) [reply doc/source/upstream-bug-fixes.rst]: Highlighted trailing whitespaces here are just a diff visual effect - the whole table is filled with spaces)

File: doc/source/upstream-bug-fixes.rst (hunk 3 above)
Comment (fiowro) [reply doc/source/upstream-bug-fixes.rst]: Same answer on highlighted trailing whitespaces (just a diff visual highlighting; the whole table is filled with spaces)

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**